src/commands/diff.ts

```diff
@@ -76,6 +76,12 @@ export default class Diff extends Command {
     checkAndWarnFalseFlag(outputFormat, markdownSubtype);
     markdownSubtype = setDefaultMarkdownSubtype(outputFormat, markdownSubtype) as string;
 
+    this.metricsMetadata.output_format = outputFormat;
+    this.metricsMetadata.output_type = outputType;
+    if (outputFormat === 'md') {
```

Comment:
Yeah, the different output formats to choose are `json`, `yaml`, `yml` and `md`. In case of selecting `md`, then we also can select a subtype: `json`, `yaml` or `yml`.

Reply:
We just want to track for `md` ?

Reply:
Not exactly. The basic metrics I think we should collect from this command are:

- `output_format`: ['json', 'yaml', 'yml', 'md']
- `output_type`: ['breaking', 'non-breaking', 'unclassified', 'all']

Then I've decided to add another metric, `output_markdown_subtype`, which only makes sense when the output format selected is 'md', as you can see on the flag definition: https://github.com/asyncapi/cli/blob/aaa8d8e989d82b3c83395bc5125e0ab8387df829/src/commands/diff.ts#L41-L45

Reply:
Then I think this code portion can be generic

Reply:
Should we discard the metric `output_markdown_subtype` then?

Reply:
No just got it now, in case of md you have subtypes, it's fine for me